### ISSUE TITLE
Decide on a dependency management system.

### ISSUE THREAD

**Issue by deanberris** [cpp-netlib/netlib#2]
Decide on using one of:

- conan.io
- vcpkg
- git submodules

for dependency management.

**Comment by glynos** [comment]
We may need to provide support for one or more.

I don't think anyone is use realistically using git submodules for dependency management for anything apart from the smallest systems.

**Comment by deanberris** [comment]
Agreed -- the question I think is whether we want to also actively own the deployment/release into those package management systems, or whether we want to require developers use one or both of them while developing?

**Comment by brown2rl** [comment]
We could own that by writing a management program/script to pull the modules from the CI server/repo.  Additionally, that management program could provide other utility commands like local testing/QA, pushing to the CI server, and building.

**Comment by deanberris** [comment]
@brown2rl -- that makes a lot of sense! Is there an existing automation process/framework we can follow for these kinds of process automation? I must admit I'm not that in-tune with the current DevOps stories/tools and would like to learn. 😄

**Comment by brown2rl** [comment]
@deanberris It looks like you went with vcpkg.

**Comment by AngleNet** [comment]
Vote on vcpkg :+1:
